launcher/src/main/java/org/wso2/carbon/launcher/config/CarbonLaunchConfig.java

```diff
@@ -79,15 +79,7 @@
      * Load the launch configuration from the classpath.
      */
     public CarbonLaunchConfig() {
-        loadFromClasspath();
-        if (logger.isLoggable(Level.FINE)) {
-            logger.log(Level.FINE, "Loaded properties from the launch.properties file.");
-            for (Map.Entry entry : properties.entrySet()) {
-                logger.log(Level.FINE, "Key: " + entry.getKey() + " Value: " + entry.getValue());
-            }
-        }
-
-        initializeProperties();
+        loadCarbonConfiguration(null);
```

Comment:
Wouldn't it be nice to have an overloaded method of "loadCarbonConfiguration" that doesn't take any parameter in?